flashinfer/moe_ep/kernel_src/cutedsl_megamoe/src/moe_nvfp4_swapab/mega_runner.py

```
@@ -399,15 +297,19 @@ def __post_init__(self) -> None:
             )
         if self.gate_up_clamp is not None and self.gate_up_clamp < 0.0:
             raise ValueError(
-                f"gate_up_clamp must be None or non-negative, got {self.gate_up_clamp}."
+                f"gate_up_clamp must be None or non-negative, got "
+                f"{self.gate_up_clamp}."
             )
 
     def __str__(self) -> str:
         dtype_name = lambda t: str(t).split(".")[-1]
```

Ruff (0.15.21)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `dtype_name` as a `def`

(E731)